js/layout-javascript/simple-list-code.js

```diff
@@ -1205,6 +1205,16 @@ DynamicList.prototype.renderLoopHTML = function (iterateeCb) {
         requestAnimationFrame(render);
       } else{
         _this.$container.find('.simple-list-container').removeClass('loading').addClass('ready');
+
+        $('[data-line-clamp]').each(function() {
+          var lineClamp = $(this).data('line-clamp');
+          var showLines =  Modernizr.ie11
+          ? lineClamp + 1 
```

Comment:
Missing indentation.

```suggestion
            ? lineClamp + 1 
```